mubench.pipeline/data/runner_interface.py

```diff
@@ -198,4 +197,4 @@ def version():
 
     @staticmethod
     def changelog():
-        return ""  # only changed Java-side interface
+        return "Only changed Java-side interface."
```

Comment:
I changed this to an empty string on purpose, because there is nothing in that version that impacts the invocation of the detector, hence, no reason to output a warning. This means we need to exclude interfaces with an empty change log when computing whether to output a warning and when computing the warning itself.